packages/frontend/core/src/modules/cmdk/entities/quick-search-manager.ts

```diff
@@ -0,0 +1,192 @@
+import type {
+  DocRecord,
+  DocsService,
+  WorkspaceLocalState,
+  WorkspaceService,
+} from '@toeverything/infra';
+import { Entity, LiveData } from '@toeverything/infra';
+import type { Observable } from 'rxjs';
+import { race, skip, Subject } from 'rxjs';
+
+import { resolveLinkToDoc } from '../../navigation';
+
+type QuickSearchMode = 'cmdk' | 'search-callback';
```

Comment:
I think naming it directly as the type to be searched is more intuitive.

```suggestion
type QuickSearchMode = 'commands' | 'docs';
```